example-code/snippets/src/clients/Events.ts

```diff
@@ -1,63 +0,0 @@
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-
-import { AgentAuthorizationClient, BackendAuthorizationClientConfiguration } from "@bentley/backend-itwin-client";
-import { AccessToken, Guid, GuidString, Logger } from "@itwin/core-bentley";
-import { EventSAS, EventSubscription, IModelHubClient, IModelHubEvent } from "@bentley/imodelhub-client";
-
-if (process.env.IMJS_AGENT_TEST_CLIENT_ID === undefined)
-  throw new Error("Could not find IMJS_AGENT_TEST_CLIENT_ID");
-if (process.env.IMJS_AGENT_TEST_CLIENT_SECRET === undefined)
-  throw new Error("Could not find IMJS_AGENT_TEST_CLIENT_SECRET");
-if (process.env.IMJS_OIDC_BROWSER_TEST_SCOPES === undefined)
-  throw new Error("Could not find IMJS_OIDC_BROWSER_TEST_SCOPES");
-
-const clientConfig: BackendAuthorizationClientConfiguration = {
-  clientId: process.env.IMJS_AGENT_TEST_CLIENT_ID ?? "",
-  clientSecret: process.env.IMJS_AGENT_TEST_CLIENT_SECRET ?? "",
-  scope: process.env.IMJS_OIDC_BROWSER_TEST_SCOPES ?? "",
-};
-
-const authorizationClient = new AgentAuthorizationClient(clientConfig);
-const imodelHubClient: IModelHubClient = new IModelHubClient();
-const imodelId: GuidString = Guid.createValue();
-
-// __PUBLISH_EXTRACT_START__ EventHandler.createListener.authenticate.example-code
-async function authenticate(): Promise<AccessToken | undefined> {
-  return authorizationClient.getAccessToken();
-}

```

Comment:
@calebmshafer  Looked like several of these examples were build around the authorizationClient. Should I just keep this whole file deleted (and delete these example code usages in Events.md) or should is there a specific way you'd recommend changing this?